src/lib/viewers/doc/_theme.scss

```diff
@@ -0,0 +1,19 @@
+@import '~box-ui-elements/es/styles/constants/colors';
+
+.custom-theme {
+    .bp .bp-thumbnail.bp-thumbnail-is-selected {
+        .bp-thumbnail-nav {
+            box-shadow: 0 0 0 3px $bdl-gray, 0 4px 5px 0 rgba(0, 0, 0, .08);
```

Comment:
Should the offset-y and blur radius for the second shadow be variables since they're shared in multiple places?

Reply:
Discussed offline.